docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -1442,12 +1447,11 @@ the following:
            of the compressed stream filled with zeros (if the
            fill bits are not zero, then the stream should be
            rejected as invalid)
-   2 bits: MNIBBLES, # of nibbles to represent the uncompressed
-           length, encoded as follows: if set to 3, MNIBBLES is 0,
-           otherwise MNIBBLES is the value of this field plus 4.
-           If MNIBBLES is 0, the meta-block is empty, i.e. it does
-           not generate any uncompressed data. In this case, the
-           rest of the meta-block has the following format:
+   2 bits: MNIBBLES - 4, # of nibbles to represent the uncompressed
```

Comment:
Sounds good. The explicit table makes it really clear. My first implementation bug was that I was checking the value of MNIBBLES prior to the add.